code/modules/antagonists/blob/blobstrains/regenerative_materia.dm

```diff
@@ -7,7 +7,7 @@
 	complementary_color = "#AF7B8D"
 	message_living = ", and you feel <i>alive</i>"
 	reagent = /datum/reagent/blob/regenerative_materia
-	core_regen_bonus = 20
+	core_regen_bonus = 18
```

Comment:
this was 20 before the linked pr was made, but it also changed point_rate_bonus from 3 to 1

Reply:
A blob regenerates using their base regen (2) added to their core regen bonus. So it was 20 before the PR, 22 after, and this brings it back down to 22. The point regen bonus is the same story, but they accounted for that one

Reply:
![image](https://user-images.githubusercontent.com/83368538/123170798-f0f11f80-d448-11eb-80f3-8eab512ca69f.png)
For reference